equinox/nn/pool.py

```diff
@@ -18,6 +18,7 @@ class Pool(Module):
     kernel_size: Union[int, Sequence[int]] = static_field()
     stride: Union[int, Sequence[int]] = static_field()
     padding: Union[int, Sequence[int], Sequence[Tuple[int, int]]] = static_field()
+    use_ceil: bool = static_field()
```

Comment:
In passing, I've just noticed that some of these type annotations are too broad. It should be:
```python
kernel_size: Sequence[int]
stride: Sequence[int]
padding: Sequence[Tuple[int, int]]
```
since we convert them from these other forms during `__init__`.